### PR TITLE
Feature/new order button

### PR DESCRIPTION
**Pull request by AnastasiiaAlyaseva**
new button with a pop-up alert

### CODE REVIEW

File: OrderForm/ContentViewModel.swift [hunk 1]
```diff
@@ -8,6 +8,16 @@ class ContentViewModel: ObservableObject {
     @Published var quantity = 1
     var pizzaOption = ["Large", "Medium"] // TODO: as smiles, but it should be connected with pizzaOptionTag
     @Published var orderStatus: Bool = false
+    @Published var showingAlert = false
```
Comment (salyasev) [inline OrderForm/ContentViewModel.swift]: showingAlert logic connected only with reset logic - may be better to move it to `ResetButtonSection`

File: OrderForm/ResetButtonSection.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+struct ResetButtonSection: View {
+    @ObservedObject var viewModel: ContentViewModel
+    
+    var body: some View {
+        Section{
+            Button ("New order") {
+                viewModel.showingAlert = true
+            }
+            .alert(isPresented:$viewModel.showingAlert) {
```
Comment (salyasev) [inline OrderForm/ResetButtonSection.swift]: code style issues

File: OrderForm/ContentViewModel.swift (hunk 1 above)
Comment (AnastasiiaAlyaseva) [reply OrderForm/ContentViewModel.swift]: corrected

File: OrderForm/ResetButtonSection.swift (hunk 2 above)
Comment (AnastasiiaAlyaseva) [reply OrderForm/ResetButtonSection.swift]: corrected

**Review by salyasev — APPROVED** [review]

**Pull request merged**